documents/plan.md

```diff
@@ -2,10 +2,27 @@
 
 ## Language evaluation
 
+The three main goals and their deliverables for this language are as follows:
+  * Develop plan and start usage of a graphics engine for the simulation
+      * Pick a graphics engine
+      * Demonstrate knowledge of the api/development aspects that can connect to my language. 
```

Comment:
A bit more detail here could be useful. For example, have you already looked at some engines; or how would you decide which engine to use (e.g., what are the properties that make an engine suitable / good for this project)?